projects/storefrontlib/src/cms-components/navigation/footer-navigation/scroll-to-top/scroll-to-top.component.html

```diff
@@ -0,0 +1,10 @@
+<!--Scroll to top-->
+<div
+  class="cx-main"
```

Comment:
Semantically, this classname should be different and more meaningful, eg. `cx-scroll-to-top-container` or something similar

Reply:
Also, if you use BEM-like naming convention, you should follow it correctly: 
- `cx-button-scroll-to-top` should be `cx-scroll-to-top-button`, etc

(applies to the whole file)